src-tauri/src/mod_manager.rs

```diff
@@ -66,8 +116,10 @@ pub fn remove_mod(mod_struct: Mod) {
 }
 
 pub(crate) fn make_tmm_game_directories(game: Game) {
-  fs::create_dir_all(game.work_path).unwrap();
-  fs::create_dir_all(game.stage_path).unwrap();
+  fs::create_dir_all(&game.profile_path).unwrap();
+  fs::create_dir_all(&game.profile_path.join("work")).unwrap();
```

Comment:
Work directory must be in the same mount device as the game's file.

Reply:
Ah, I didn't know that. Will change that as soon as I can!

Reply:
> Ah, I didn't know that. Will change that as soon as I can!

So more info on this, if the game is located in /mnt/id/something then the work directory needs to be in a writable location in /mnt/id/ 

Since games can be installed on any drive what i did is put the work directory in the same location as the steam library directory since you know that if the steam library is there it is writable and you also know that it is on the same mount device as the game

Reply:
Please look at how i do it currently work_path in mod_manager.rs it was working fine.

Reply:
Will do!